test/e2e/e2e_test.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/kubernetes-sigs/kubebuilder/test/e2e/framework"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	framework.RegisterFlags()
+	pflag.Parse()
+}
```

Comment:
We should avoid code in the `init` block if possible.

Reply:
I'm afraid we have to make the flag things in the `init` block, or the flags we passed in will not be recognized.